lighthouse-core/test/lib/i18n/swap-locale-test.js

```diff
@@ -26,6 +26,12 @@ describe('swap-locale', () => {
     // Renderer formatted strings
     expect(lhrEn.i18n.rendererFormattedStrings.labDataTitle).toEqual('Lab Data');
     expect(lhrEs.i18n.rendererFormattedStrings.labDataTitle).toEqual('Datos de prueba');
+
+    // Formatted numbers in placeholders.
+    expect(lhrEn.audits['render-blocking-resources'].displayValue)
+      .toEqual('Potential savings of 1,130 ms');
+    expect(lhrEs.audits['render-blocking-resources'].displayValue)
+      .toEqual('Potential savings of 1.130 ms');
```

Comment:
this is actually formatted by `intl-messageformat`, but it's one more check that the number locales are being initialized correctly and it seemed like a missing piece of checking locale swapping